pubspec.yaml

```diff
@@ -29,38 +29,29 @@ dependencies:
     git:
       url: https://github.com/google/flutter-desktop-embedding.git
       path: plugins/window_size
-  basic_utils:
-    git:
-      url: https://github.com/Ephenodrom/Dart-Basic-Utils
+      ref: 17d4710c17f4913137e7ec931f6e71eaef443363
   file_picker: ^4.5.0
   intro_slider:
     git:
       url: https://github.com/shareef-dweikat/flutter-intro-slider
+      ref: a9bb492412c68419900750fe12e0487e3a1168fb
```

Comment:
can we do releases of these? or at least named tags (since they appear to be repos "we" control?)

Reply:
That's reasonable, I agree, CC @shareef-dweikat 

I put the hashes here as an improvement over nothing at all. I guess also tags may be overwritten to point to different hashes. In that sense, maybe hashes are more reproducible?

Reply:
Yeah, I guess depends what the use-case is ... for reproducibility I suppose the hash is better here (coming from python-land, you'd pin against "the hash of the 1.2.3 release on PyPI" normally). 

Even if _this_ uses the hash it could still be beneficial to have like "22.9.1" (date-based) tags to help the humans but I'll leave that to people actually doing this ;) and also I don't know flutter ecosystem at all